guides/installation/requirements.md

```diff
@@ -55,7 +55,7 @@ You can use these commands to check your actual environment:
 
 * MySQL
 
-  * Recommended version: 8.0
+  * Compatible versions: 8.0 and 9.0
```

Comment:
MySQL 9 is an innovation release. I would add something like not recommended to use in production or even tbh not mention here 🤔

Reply:
I tend to agree, maybe we could ignore listing it here, and if someone asks specifically, we could tell them that it should be fine but not recommended

Reply:
@Gaitholabi or @shyim , shall I close this PR or change "recommended version to compatible version" ?

Reply:
@sushmangupta i'll close it for now 👍